Adding 'How Flexible Are You?' Post

nickloestartup opened the pull request:
Grateful if someone can run this locally to make sure there are no formatting gremlins.
### Blog title

How Flexible Are You?
### Blog description

How flexible does a Scrum Master need to be?
#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge
#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge
#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

christinamattinson commented on _posts/2016-10-07-how-flexible-are-you.md:
```diff
@@ -0,0 +1,43 @@
+This is the definition of agile from [Oxford Dictionaries](http://www.oxforddictionaries.com/definition/english/agile):
+>“Able to move quickly and easily” and “Relating to or denoting a method of project management, used especially for software development, that is characterized by the division of tasks into short phases of work and **frequent reassessment and adaptation of plans**”
+
+Given that Scrum is an Agile framework, it shouldn't be surprising if I said a Scrum Master had to be agile, or flexible. In fact flexibility is at the core of a Scrum Master’s role. But flexibility is multifaceted when it comes to being a Scrum Master. Since [Holiday Extras](http://www.holidayextras.co.uk/) divided our Web Team resources into Pods (our small, cross-functional, Agile teams), Scru ...
```
I would avoid referring to the team as resources, just team or people sounds better :)

christinamattinson commented:
Lots of great examples of flexibility, I think at times the wording could be more concise and I know the blog is about flexibility - but in some areas I wonder if you could use another word, Maybe proactive to change, embracing change to continuously improve, creativity and empowerment to try out new things like retros etc.

christinamattinson commented:
Taking one spot :)

nickloestartup commented on _posts/2016-10-07-how-flexible-are-you.md:
```diff
@@ -0,0 +1,43 @@
+
+Flexitime is a luxury the whole Web Team is entitled to and certainly embraces. Although our working days accomodate our personal lives, our productivity remains strong. This is in part due to the great and mature attitude of the team, but also credit to Scrum Masters who seamlessly adapt to support colleagues whose hours are not your average 9-5.
+
+We demonstrate flexibility to support resource changes. Our development teams are mainly made up of Web Engineers and Software Testers, and we frequently move people around Pods where the business dictates they need to be, or for their own personal development. It certainly keeps everyone on their toes and these changes require flexibility from Scrum Masters to adapt, teach, coach and support, wh ...
```
Thanks, I've reworded.

nickloestartup commented:
@christinamattinson thanks for taking a look and for the comments. I've addressed the 2 comments above. On the overuse of flexible/ flexibility, I debated whether to go heavy or not on this as it wasn't reading right to me. But I felt I'd have to rewrite a huge amount to change it. However, given your comments I agree it's the right way to go. As a result I've gone through and changed a lot and had to rewrite to accommodate these changes. 

Grateful if you can take another read through :-)

alannajenkins commented:
Taking second editor.

alannajenkins commented on _posts/2016-10-07-how-flexible-are-you.md:
```diff
@@ -0,0 +1,43 @@
+
+There are many areas to a Scrum Master role and the ability to switch between them is fundamental to a Scrum Master’s success, and that of their Pod. We aim to create a barrier to our development teams to avoid distractions, act as facilitators, mediators, coaches and multi-task with many competing priorities.
+
+So flexibility is key. But not just to switch between competing demands, but also to be versatile in the way we Scrum. We’re given the freedom at HX to tailor the Scrum process so that it works best for us as individuals, the Product Owners we support and, as importantly, the development teams in our Pods. We want to gain their trust and commitment, to promote a culture where open discussion and  ...
```
would maybe reword the last sentence in this paragraph to say "This is really important for the productivity in our Pods, and for keeping them focussed on adding maximum value for our customers."

alannajenkins commented on _posts/2016-10-07-how-flexible-are-you.md:
```diff
@@ -0,0 +1,43 @@
+
+With creative licence to run our Sprints how we want to, no Sprint Planning, Retrospective, Stand-up or Sprint Review is facilitated identically from Scrum Master to Scrum Master. It’s credit to how adaptable our Pod members are that they embrace these variations (in part they are tailored to those Pod members, however, they also largely account for an individual Scrum Master’s style). That freed ...
+
+In recent months we have started to work with some Pod members remotely. This is in part due to our expansion into a tech hub in Sofia, Bulgaria, however, it’s also in recognition of the flexible way we approach work. Be it from the comfort of our own homes, in shared workspaces, collaborative working spaces at employees’ homes or now hubs in other countries, remote working is increasingly part o ...
```
there's a word missing in your third sentence - should read "remote working is increasingly a part of the way we work"

alannajenkins commented on _posts/2016-10-07-how-flexible-are-you.md:
```diff
@@ -0,0 +1,43 @@
+
+Flexitime is a luxury the whole Web Team is entitled to and certainly embraces. Although our working days accomodate our personal lives, our productivity remains strong. This is in part due to the great and mature attitude of the team, but also credit to Scrum Masters who seamlessly adapt to support colleagues whose hours are not your average 9-5.
+
+We embrace fluidity of the people in our teams, as individuals move to supports business goals and development opportunities. It definitely keeps everyone on their toes and scrum masters take these opportunities to adjust, teach, coach and support, to make these transitions smooth and continue seamless delivery.
```
"We embrace the fluidity of the people in our teams"?

alannajenkins replied:
supports doesn't need the s on the end, as it's already on the end of individuals.

alannajenkins replied:
In the second line, scrum masters in written in lower case, whereas everywhere else so far it's been written as Scrum Masters. For consistency in formatting, it would be worth capitalising this mention.

nickloestartup replied:
Have reworded this

nickloestartup replied:
👍

christinamattinson commented:
I have re read and it's a great improvement Nick. 👍 My phone won't let me tick the boxes but good to go after Alanna's comments.

nickloestartup commented:
@alannajenkins thanks for the comments. I've made the suggested changes and also reworded that one sentence - does it make more sense now? and are you happy to also 👍 it?

alannajenkins commented on _posts/2016-10-07-how-flexible-are-you.md:
```diff
@@ -0,0 +1,43 @@
+
+In recent months we have started to work with some Pod members remotely. This is in part due to our expansion into a tech hub in Sofia, Bulgaria, however, it’s also in recognition of the flexible way we approach work. Be it from the comfort of our own homes, in shared workspaces, collaborative working spaces at employees’ homes or now hubs in other countries, remote working is increasingly a part ...
+
+Flexitime is a luxury the whole Web Team is entitled to and certainly embraces. Although our working days accomodate our personal lives, our productivity remains strong. This is in part due to the great and mature attitude of the team, but also credit to Scrum Masters who seamlessly adapt to support colleagues whose hours are not your average 9-5.
```
accommodate has two m's

nickloestartup replied:
Thanks. Fixed.

alannajenkins commented:
Changes are all good with me. I've just spotted one spelling mistake that I missed on first read. Once that's sorted, then I'm happy with everything.
